packages/phone-number-privacy/signer/src/server.ts

```diff
@@ -20,7 +20,7 @@ export enum Endpoints {
 export function createServer() {
   logger.info('Creating express server')
   const app = express()
-  app.use(express.json(), loggerMiddleware)
+  app.use(express.json({ limit: '0.2mb' }), loggerMiddleware)
```

Comment:
There was a `TODO` on handling reasonably sized request body's. I agree that JSON.stringify'ing isn't a great solution, so instead of handling it manually (something like checking `req.rawBody.length`), we can just offload this logic to Express.   

Unfortunately (AFAICT) there's no such check we can add in firebase-functions, and they don't expose the `req.rawBody` property.

Reply:
Sounds good!